contracts/patterns/Upgradeable.sol

```diff
@@ -4,6 +4,8 @@
 
 pragma solidity >=0.6.0 <0.9.0;
 
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
```

Comment:
Why do we need to import `ERC165`?

Reply:
Weird, but yes, otherwise hardhat compile upon package prepare fails